en/manual/05-nodes.md

```diff
@@ -129,4 +129,4 @@ The example below shows that the user can't create Job for those nodes.
 [dispatch]
   ~ List nodes or execute commands and scripts.
 
-[dispatch]: ../man1/dispatch.html
+[dispatch][page:manpages/man1/index.md]
```

Comment:
Remove this, replace earlier `[dispatch]` link with this content

Reply:
fixed...